remote-processor/Socket.cpp

```diff
@@ -108,7 +108,7 @@ bool CSocket::read(void* pvData, uint32_t uiSize)
 
     while (uiSize) {
 
-        int32_t iAccessedSize = ::recv(_iSockFd, &pucData[uiOffset], uiSize, MSG_NOSIGNAL);
+        int32_t iAccessedSize = ::recv(_iSockFd, &pucData[uiOffset], uiSize, 0);
```

Comment:
`flags` is an int and from `man recv`:

    The flags argument is formed by ORing one or more of the following values:

So, `0` is correct.